library_generation/cli/entry_point.py

```diff
@@ -88,6 +88,20 @@ def generate(
     The commit history, if generated, will be available in
     repository_path/pr_description.txt.
     """
+    generate_impl(
+        baseline_generation_config_path, current_generation_config_path, repository_path
+    )
+
+
+def generate_impl(
```

Comment:
Prefix this method with `__` to indicate this is a private method?

Reply:
Yes, it's indeed a private method. Done, and renamed the method as discussed internally.